src/librbd/ManagedLock.h

```diff
@@ -170,18 +215,18 @@ class ManagedLock {
   std::string m_cookie;
   std::string m_new_cookie;
 
+  State m_state;
```

Comment:
@dillaman Just noticed, there are compilation warnings due to wrong initialization order in the constructor